templates/output.html.twig

```diff
@@ -23,6 +23,12 @@
                         {{ msg('engine-name-tesseract') }}
                     </label>
                 </div>
+                <div class="radio">
+                    <label>
+                        <input type="radio" name="engine" value="transkribus" {% if engine == 'transkribus' %}checked{% endif %} />
+                        {{ msg('engine-name-transkribus') }}
+                    </label>
+                </div>
```

Comment:
This UI is pretty simple, so it's good to have it here. But note that this task is about adding to the API so should also have additional edits to the OpenApi docs for the `apiAction()` method in the controller.